components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/config/TOMLConfigParser.java

```diff
@@ -81,4 +81,5 @@ public static void write(String configFilePath, Object content) throws ConfigPar
     private static <T> T parseString(String configFileContent, Class<T> type) {
         return new Toml().read(configFileContent).to(type);
     }
+
```

Comment:
shall we remove this blank line

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/332/commits/7dec453bb0eb0d526d5b95004e51f7b012efb678